app/models/kms/page.rb

```diff
@@ -55,7 +55,7 @@ def fetch_item!(slug)
     end
 
     def register_id
-      "#{self.class.name.parameterize("_")}_#{id}"
+      (self.class.name << "_" << self.id.to_s).parameterize(separator: "_")
```

Comment:
there are actually 2 place where I changed it and reason is this
<img width="969" alt="screen shot 2017-01-22 at 12 21 38" src="https://cloud.githubusercontent.com/assets/342814/22182460/685021f6-e09d-11e6-9ae7-79966680e9f0.png">